src/praisonai-bot/praisonai_bot/gateway/push_delivery.py

```
@@ -85,10 +202,24 @@ async def stop_sweeper(self) -> None:
+        # Durable by default: persist to the local SQLite store so pending
+        # events survive a gateway restart with no external service.
+        if self._sqlite_store is not None:
+            try:
+                await asyncio.get_event_loop().run_in_executor(
+                    None,
+                    self._sqlite_store.put,
+                    event.event_id,
+                    event.to_dict(),
+                    event.timestamp,
+                )
+            except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)